jenkins_pipelines/environments/common/pipeline-reference.groovy

```diff
@@ -3,6 +3,11 @@ def run(params) {
         deployed = false
         env.resultdir = "${WORKSPACE}/results"
         env.resultdirbuild = "${resultdir}/${BUILD_NUMBER}"
+
+        if (!params.terraform_parallelism) {
```

Comment:
I'm not sure if in groovy that's the way to handle an element not defined.
If that's ok, then good:)

Reply:
Jep, groovy is able to handle this. I looked at some other examples. Jordi also commented below.